app/api/events.py

```diff
@@ -70,6 +70,13 @@ def validate_event(user, modules, data):
         raise ForbiddenException({'source': ''},
                                  "Only verified accounts can publish events")
 
+    if data.get('state', None) == 'published':
+        event = Event.query.filter_by(name=data.get('name', None)).first()
+        ticket_available = event.tickets_available
+        if (ticket_available == 0):
```

Comment:
You could make this as 
```
if event.tickets:
    ....
```

Reply:
I agree to @mrsaicharan1. We already face this a lot, you need to check if anything exists first then try to retrieve it

Reply:
I will make appropriate changes